test/sparsedir/sparsevector.jl

```diff
@@ -868,6 +868,28 @@ let m = 10
     end
 end
 
+# fkeep!
+let x = sparsevec(1:7, [3., 2., -1., 1., -2., -3., 3.], 7)
+    # droptol
+    xdrop = Base.droptol!(copy(x), 1.5)
+    @test exact_equal(xdrop, SparseVector(7, [1, 2, 5, 6, 7], [3., 2., -2., -3., 3.]))
+    Base.droptol!(xdrop, 2.5)
+    @test exact_equal(xdrop, SparseVector(7, [1, 6, 7], [3., -3., 3.]))
```

Comment:
could you add a test of `droptol!` where the tolerance is exactly equal to one of the nonzero values? then I think this is done